lib/commit_after_review.js

```diff
@@ -0,0 +1,73 @@
+function commitAfterReview(commit, review, logger) {
+  let commitIndex = commit.length === 1
+    ? 0 : commit.length - 1;
+  let reviewIndex = review.length === 1
+    ? 0 : review.length - 1;
+
+  let lastCommit = commit[commitIndex].commit;
+  let lastReview = review[reviewIndex];
+
+  let commitInfo = getDate(lastCommit.committedDate);
+  let reviewInfo = getDate(lastReview.publishedAt);
+
+  switch (false) {
+    // very edge case senario
+    // commit after 1 year!
+    case (commitInfo.year < reviewInfo.year ||
+          commitInfo.year === reviewInfo.year):
+      warn();
+      break;
+
+    case (commitInfo.month < reviewInfo.month ||
+          commitInfo.month === reviewInfo.month):
+      warn();
+      break;
+
+    case (commitInfo.date < reviewInfo.date ||
+          commitInfo.date === reviewInfo.date):
+      warn();
+      break;
+
+    case (commitInfo.hour < reviewInfo.hour ||
+          commitInfo.hour === reviewInfo.hour):
+      warn();
+      break;
+
+    case (commitInfo.minutes < reviewInfo.minutes ||
+         commitInfo.minutes === reviewInfo.minutes):
+      warn();
+      break;
+  }
+
+  function warn() {
+    let msg = 'There has been new commits pushed' +
+              ' to Pull Request the since last review';
+    logger.warn(msg);
+    let pullURL = lastReview.url.split('#')[0];
+    let commitURL = `${pullURL}/commits/${lastCommit.oid}`;
+    let allChangesURL = `${pullURL}/files`;
+    logger.warn('New commit url: ', commitURL);
+    logger.warn('see all the changes: ', allChangesURL);
```

Comment:
I don't think this link is particularly helpful as it does not show any correlation between reviews, commits and changes.

Reply:
I would probably just add the link to the list of commits instead of the above lines, we need to check this manually anyway.